lib/main_micro_alp.pm

```diff
@@ -321,13 +323,25 @@ sub load_slem_on_pc_tests {
                 $run_args->{runtime} = $_;
                 load_container_engine_test($run_args);
             }
-
             loadtest("publiccloud/ssh_interactive_end", run_args => $args);
-        }
-        else {
+        } else {
             loadtest "publiccloud/check_services", run_args => $args;
             loadtest("publiccloud/slem_basic", run_args => $args);
         }
+        # AISTACK test verification
+        if (get_var('PUBLIC_CLOUD_AISTACK')) {
+            # loadtest("publiccloud/ssh_interactive_start", run_args => $args);
+            # loadtest("publiccloud/instance_overview", run_args => $args);
+            # loadtest("publiccloud/slem_prepare", run_args => $args);
+            my $runtime = get_required_var('CONTAINER_RUNTIMES');
+            for (split(',\s*', $runtime)) {
```

Comment:
I assume this loop can be dropped because you have another one which you run within `aistack_basic`

Reply:
this is the one which loops thru and installs the required dependency. inside the aistack_basic just double check the package installed.

Reply:
this is NOT function which you need to use to just install dependencies . the fact that dependencies are installed is just side effect of this function which is doing a lot of other things on top . Please find other way to install all needed dependencies

Reply:
Removed and included dep pkg install in aistack_basic.pm